cfgov/ask_cfpb/models/pages.py

```diff
@@ -869,3 +871,145 @@ def meta_image(self):
+
+
+class ArticleLink(Orderable, models.Model):
+    text = models.CharField(max_length=255)
+    url = models.CharField(max_length=255)
+
+    article_page = ParentalKey(
+        'ArticlePage',
+        related_name='article_links'
+    )
+
+    panels = [
+        FieldPanel('text'),
+        FieldPanel('url'),
+    ]
+
+
+class ArticlePage(CFGOVPage):
+    """
+    General article page type.
+    """
+    category = models.CharField(
+        choices=[
+            ('basics', 'Basics'),
+            ('common_issues', 'Common issues'),
+            ('howto', 'How to'),
+            ('know_your_rights', 'Know your rights'),
+        ],
+        max_length=255,
+    )
+    heading = models.CharField(
+        max_length=255,
+        blank=False,
+    )
+    intro = models.TextField(
+        blank=False
+    )
+    inset_heading = models.CharField(
+        max_length=255,
+        blank=True,
+        verbose_name="Heading"
+    )
+    sections = StreamField([
+        ('section', blocks.StructBlock([
+            ('heading', blocks.CharBlock(
+                max_length=255,
+                required=True,
+                label='Section heading'
+            )),
+            ('summary', blocks.TextBlock(
+                required=False,
+                blank=True,
+                label='Section summary'
+            )),
+            ('link_text', blocks.CharBlock(
+                required=False,
+                blank=True,
+                label="Section link text"
+            )),
+            ('url', blocks.CharBlock(
+                required=False,
+                blank=True,
+                label='Section link URL',
+                max_length=255,
+            )),
+            ('subsections', blocks.ListBlock(
+                blocks.StructBlock([
+                    ('heading', blocks.CharBlock(
+                        max_length=255,
+                        required=False,
+                        blank=True,
+                        label='Subsection heading'
+                    )),
+                    ('summary', blocks.TextBlock(
+                        required=False,
+                        blank=True,
+                        label='Subsection summary'
+                    )),
+                    ('link_text', blocks.CharBlock(
+                        required=True,
+                        label='Subsection link text'
+                    )),
+                    ('url', blocks.CharBlock(
+                        required=True,
+                        label='Subsection link URL'
+                    ))
+                ])
+            ))
+        ]))
+    ])
+    content_panels = CFGOVPage.content_panels + [
+        MultiFieldPanel([
+            FieldPanel('category'),
+            FieldPanel('heading'),
+            FieldPanel('intro')],
+            heading="Heading",
+            classname="collapsible"),
+
+        MultiFieldPanel([
+            FieldPanel('inset_heading'),
+            InlinePanel(
```

Comment:
It might be nice to include a `help_text` value to the InlinePanel to describe where an Inset link is going to appear, and maybe even roughly what it's going to look like, though that may be overkill.

Reply:
@higs4281 That's a really good point. I think there's a demo/user test coming up with CE editorial in the next few weeks, so maybe we can plan to add help text to this and other fields after their feedback.